Relation-SPA/src/app/relation-table/relation-table.component.html

```diff
@@ -62,6 +57,9 @@
       </tr>
     </tbody>
   </table>
+  <pagination [boundaryLinks]="true" [totalItems]="77"
```

Comment:
Why [totalItems]="77"? Probably add a comment in the code - "TODO: ......."

Reply:
Proper ToDo comment was added to mark feature as ToDo;